test/e2e/view-caseload-progress.js

```diff
@@ -0,0 +1,27 @@
+const expect = require('chai').expect
+
+const workloadCapacityHelper = require('../helpers/data/workload-capacity-helper')
+
+var workloadOwnerIds = []
+
+describe('View caseload progress flow', () => {
+  before(function () {
+    return workloadCapacityHelper.selectIdsForWorkloadOwner()
+      .then(function (results) {
+        workloadOwnerIds = results
+      })
+  })
+
+  it('should navigate to the workload owner caseload progress screen', () => {
+    return browser.url('/offender-manager/' + workloadOwnerIds.filter((item) => item.table === 'workload_owner')[0].id + '/caseload-capacity')
+      .waitForExist('.breadcrumbs')
+      .waitForExist('.c-subNav')
```

Comment:
after chatting with @stevenadams we're going to update this to `.wmt-sub-nav` for now, but it shouldn't block the story from being signed off so we'll cover it off with an async clean up task.